scripts/legislation/africa/malawi_english.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^ ENACTED by the Parliament of Malawi as follows/,$p' | \
+    sed -n '/^ PART I -/,$p'
+}
+
+function prefix_article_numbers_with_article_literal {
+  sed -E 's/^([0-9]+\.)/Article \1/'
+}
+
+function replace_parentheses_around_article_delimiters_with_square_brackets {
+  sed -E 's/^\(([A-Za-z0-9]+)\)/[\1]/' 
+}
+
+function amend_errors_in_headers {
+  sed -E 's/PART/\n\nPART/g' | \
+    sed -E 's/^(PART .*)\[•\]/\1-/' | \
+    sed -n '/PART I /,$p' | \
+    sed -E 's/Projects for which an environmen tal impact assessment is required //' 
+}
+
+function format_article_literals {
+  sed -E 's/(Article )\[([0-9]+)\] \[•\]/\n\n\(\2\)/g' | \
+    sed -E 's/\[([0-9]+)\] \[•\]/\n\n\(\1\)/g' 
+}
+
+function amend_article_numbering {
+  sed -E 's/\(16\)/(17)/' | \
+    sed -E 's/\(15\)/(16)/' | \
+    sed -E 's/\(14\)/(15)/' | \
+    sed -E 's/\(13\)/(14)/' | \
+    sed -E 's/\(12\)/(13)/' | \
+    sed -E 's/\(11\)/(12)/' | \
+    sed -E 's/\(10\)/(11)/' | \
+    sed -E 's/\(9\)/(10)/' | \
+    sed -E 's/\(8\)/(9)/' | \
+    sed -E 's/\(7\)/(8)/' 
+}
+
+function amend_errors_in_articles {
+  sed -E 's/ \[•\]/:/g' | \
+    sed -E 's/\(([a-z])\)/[\1]/g' | \
+    sed -E 's/ \(([0-9])\)/ [\1]/g' | \
```

Comment:
These two lines seem redundant (`replace_parentheses_around_article_delimiters_with_square_brackets`). Am I mistaken?

Reply:
I have implemented your suggestion as in the Uganda legislation